REL/mention_detection_base.py

```diff
@@ -39,6 +40,7 @@ def get_ctxt(self, start, end, idx_sent, sentence, sentences_doc):
 
         return left_ctxt, right_ctxt
 
+    @lru_cache
```

Comment:
Let's not use default settings for `lru_cache` as we then only cache the latest 128 calls will be cached. Perhaps we can allow for the configuration of this method? Perhaps create a config.yml file or at least make it ia static global variable.

Reply:
I agree this could be bigger, but I noticed this already helps a lot. Mostly because the same mentions often occur close to each other, because if an entity occurs in a document is often is mentioned multiple times in that document.

Reply:
So shall we change it to something higher then, or do prefer keeping it at its current default value?

Reply:
For my current experiments, this is fine. But I think it would be useful to spend some time researching the ideal size, but I don't have time to do a thorough investigation right now.